### DISCUSSION TITLE
Calendar description and HTML

### DISCUSSION THREAD

**Discussion by StephenESW** [FossifyOrg/General-Discussion#465] [Ideas]
I really love this calendar! One thing I'd like to suggest is that the calendar description could be in HTML rather than plain text. Syncing this calendar with Google calendar in Mozilla Thunderbird results in a mess because those descriptions are in HTML. Or am I missing something / not doing something right?

**Comment by Aga-C** [comment]
There's already a request for this: https://github.com/FossifyOrg/Calendar/issues/12.

**Reply by StephenESW** [reply]
My apology. I tried to search, but missed this. Thank you for finding it for me.
